src/pid.cpp

```diff
@@ -104,12 +116,12 @@ void Pid::initPid(
         }
       };
 
-    declare_param("p", rclcpp::ParameterValue(p));
-    declare_param("i", rclcpp::ParameterValue(i));
-    declare_param("d", rclcpp::ParameterValue(d));
-    declare_param("i_clamp_max", rclcpp::ParameterValue(i_max));
-    declare_param("i_clamp_min", rclcpp::ParameterValue(i_min));
-    declare_param("antiwindup", rclcpp::ParameterValue(antiwindup));
+    declare_param(parameter_prefix_ + "p", rclcpp::ParameterValue(p));
+    declare_param(parameter_prefix_ + "i", rclcpp::ParameterValue(i));
+    declare_param(parameter_prefix_ + "d", rclcpp::ParameterValue(d));
+    declare_param(parameter_prefix_ + "i_clamp_max", rclcpp::ParameterValue(i_max));
+    declare_param(parameter_prefix_ + "i_clamp_min", rclcpp::ParameterValue(i_min));
+    declare_param(parameter_prefix_ + "antiwindup", rclcpp::ParameterValue(antiwindup));
```

Comment:
I don't think that the `declare_param` calls should be part of the init function, but rather be declared within the ctor.